apps/nestjs-wallet/src/vc-api/credentials/credentials.service.spec.ts

```diff
@@ -16,6 +18,65 @@ const key = {
 };
 const did = keyToDID('key', JSON.stringify(key));
 
+const credential: CredentialDto = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    {
+      issuerFields: {
+        '@id': 'ew:issuerFields',
+        '@type': 'ew:IssuerFields'
+      },
+      namespace: 'ew:namespace',
+      role: {
+        '@id': 'ew:role',
+        '@type': 'ew:Role'
+      },
+      ew: 'https://energyweb.org/ld-context-2022#',
+      version: 'ew:version',
+      EWFRole: 'ew:EWFRole'
+    }
+  ],
+  id: 'urn:uuid:7f94d397-3e70-4a43-945e-1a13069e636f',
+  type: ['VerifiableCredential', 'EWFRole'],
+  credentialSubject: {
+    id: 'did:example:d23dd687a7dc6787646f2eb98d0',
+    issuerFields: [],
+    role: {
+      namespace: 'test.iam.ewc',
+      version: '1'
+    }
+  },
+  issuer: did,
+  issuanceDate: '2022-03-18T08:57:32.477Z'
+};
+const vc: VerifiableCredential = {
+  ...credential,
+  proof: {
+    type: 'Ed25519Signature2018',
+    proofPurpose: 'assertionMethod',
+    verificationMethod:
+      'did:key:z6MkoB84PJkXzFpbqtfYV5WqBKHCSDf7A1SeepwzvE36QvCF#z6MkoB84PJkXzFpbqtfYV5WqBKHCSDf7A1SeepwzvE36QvCF',
+    created: '2021-11-16T14:52:19.514Z',
+    jws: 'eyJhbGciOiJFZERTQSIsImNyaXQiOlsiYjY0Il0sImI2NCI6ZmFsc2V9..CTodsuvLnodd4nhlxHA39XA0BSfjf6p0rsMgApBmIMRz2MOBF2WnK_2Fd5RBJ-SxRmqvFqa-eJy1LWF74gDUBw'
+  }
+};
+const presentation: Presentation = {
+  '@context': ['https://www.w3.org/2018/credentials/v1', 'https://www.w3.org/2018/credentials/examples/v1'],
+  type: ['VerifiablePresentation'],
```

Comment:
```suggestion
  type: ['VerifiablePresentation'],
  holder: 'holder DID'
```
Presentation should have a holder property.